### ISSUE TITLE
你的效果是如何达到的?

### ISSUE THREAD

**Issue by swy0915** [mhcao916/NER_Based_on_BERT#2]
我调通了你的代码,但是一样的参数跑出来的效果没你贴出来的好.
eval_f = 0.6481409
eval_precision = 0.66809785
eval_recall = 0.64605385
global_step = 16000
loss = 0.058055636
请问你是怎么训练出来的?为什么一样的参数根本达不到你贴出来的效果?

**Comment by NexusLee** [comment]
兄嘚 我也是 而且我测试的时候都是回复 []
